core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Observable, Subject } from 'rxjs';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { FormGroup, Validators, UntypedFormBuilder } from '@angular/forms';
+import { DynamicDialogConfig, DynamicDialogRef } from 'primeng/dynamicdialog';
+import { map, startWith, takeUntil } from 'rxjs/operators';
+import { DotRole } from '@dotcms/app/shared/models/dot-role/dot-role.model';
+import { DotPageRenderState } from '../../shared/models';
+import { DotFavoritePageState, DotFavoritePageStore } from './store/dot-favorite-page.store';
+
+export interface DotFavoritePageProps {
+    pageRenderedHtml?: string;
+    pageState: DotPageRenderState;
+}
+
+export interface DotFavoritePageFormData {
```

Comment:
move these interfaces to other file ?

Reply:
this is only used on this component due to its props and form

Reply:
Then export is not need it.

Reply:
> Then export is not need it.

it's also used on the STORE (that's why the `export`), still don't think it's needed to be put on a separate file

Reply:
@oidacra and I are working in a structure for this... will be out soon.